.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+
```

Comment:
Maybe we should add some code owners here for future pull requests.

Reply:
I like the idea of using `CODEOWNERS` but I feel like its very high maintenance to keep track of, and doesn't effectively capture the transient nature of our contributors gradually gaining and removing responsibility from each repository. This file expresses ownership more like a boolean instead of like the continuum of responsibility I have seen.